bayesian/api_v1.py

```diff
@@ -1217,18 +1217,38 @@ class CveByDateEcosystem(Resource):
     method_decorators = [login_required]
 
     @staticmethod
-    def get(modified_date, ecosystem=None):
+    def get(end_date, ecosystem='all'):
         """Implement GET Method."""
-        if not modified_date:
+        if not end_date:
             raise HTTPError(400, error="Expected date in the request")
         try:
-            datetime.datetime.strptime(modified_date, '%Y%m%d')
+            datetime.datetime.strptime(end_date, '%Y%m%d')
         except ValueError:
             msg = 'Invalid datetime specified. Please specify in YYYYMMDD format'
             raise HTTPError(400, msg)
-        getcve = CveByDateEcosystemUtils(modified_date, ecosystem)
-        result = getcve.get_cves_by_date() if not ecosystem else getcve.get_cves_by_date_ecosystem()
-        return jsonify(result), 200
+        # Request Parameters Used
+        result_type = request.args.get('result_type', 'epvs')
```

Comment:
@animuk - TBH, I am not getting the point here for a single endpoint to fetch either CVE details or fetch CVE associated with EPVs. Why do you need to combine them here? Or maybe, I am not getting what you're trying to achieve here.